Student/StudentUITests/Submissions/SubmissionButton/SubmissionButtonTests.swift

```diff
@@ -22,11 +22,13 @@ import TestsFoundation
 import XCTest
 
 class SubmissionButtonTests: StudentUITestCase {
-    lazy var course: APICourse = {
-        let course = APICourse.make()
+    var course: APICourse!
```

Comment:
`let course = APICourse.make()`

Id' rather not need `!` if it's easy to avoid.